nextcord/application_command.py

```diff
@@ -237,7 +237,7 @@ def __init__(self, parameter: Parameter):
     def description(self) -> str:
         """If no description is set, it returns the bare minimum that Discord demands."""
```

Comment:
This should be changed to not if it's not provided the new return is used

Reply:
Fixed!